security/tutorials/app-layer-policy/enforce-policy-istio.md

```diff
@@ -216,10 +270,11 @@ requests.
 
 ```yaml
 apiVersion: projectcalico.org/v3
-kind: GlobalNetworkPolicy
+kind: NetworkPolicy
```

Comment:
The policy in the yaml used above are GlobalNetworkPolcy, so not sure why we are changing this to NetworkPolicy.

Reply:
fixed.